src/main/java/com/softserveinc/ita/rozetka/page_objects/BasketPage.java

```diff
@@ -0,0 +1,43 @@
+package com.softserveinc.ita.rozetka.page_objects;
+
+import com.codeborne.selenide.Condition;
+
+import static com.codeborne.selenide.Condition.not;
+import static com.codeborne.selenide.Selenide.$x;
+
+public class BasketPage {
+
+    public String getProductTitleByName(String productName) {
+        return $x(String.format("//a[@class='cart-product__title' and contains(text(), '%s')]", productName)).text();
+    }
+
+    public OrderPage openOrderPage() {
+        $x("//*[@class='button button_size_large button_color_green cart-receipt__submit']").click();
+        return new OrderPage();
+    }
+
+    public BasketPage increaseProductCount(int number) {
```

Comment:
number - number to increase or number or product?

Reply:
method increases the count of specific product selected by its number in the basket
variable (int number) changed to (int numberOfProduct)